modules/reference/pages/tunable-properties.adoc

```diff
@@ -1372,6 +1372,38 @@ Maximum size of the user-space receive buffer. If `null`, this limit is not appl
 
 ---
 
+[[kafka_throughput_throttling_v2]]
+=== kafka_throughput_throttling_v2
+
+This disables version two of throughput throttling, introduced with Redpanda v23.3.8. When this is disabled, Redpanda uses the throttling mechanism from v23 and older. This defaults to true for all new or upgraded Redpanda clusters.
```

Comment:
```suggestion
This enables version two of throughput throttling, introduced with Redpanda v23.3.8. Set to false to use the throttling mechanism from v23.3.7 and older. This defaults to true for all new or upgraded Redpanda clusters.
```
Or, from https://github.com/redpanda-data/redpanda/blob/v23.3.8/src/v/config/configuration.cc#L2792-L2793
```suggestion
Use throughput throttling based on a shared token bucket instead of balancing quota between shards.
```

Reply:
I used a little bit of both pieces of language.